server/src/service_manager.rs

```diff
@@ -1,24 +1,23 @@
 // Secret Service Manager
 
-use std::{collections::HashMap, sync::Arc};
+use std::sync::Arc;
 
 use tokio::sync::RwLock;
-use zbus::zvariant::{ObjectPath, OwnedObjectPath};
+use zbus::zvariant::ObjectPath;
 
-use crate::session::Session;
+use crate::Service;
 
 #[derive(Debug)]
 pub struct ServiceManager {
     connection: zbus::Connection,
-    // sessions mapped to their corresponding object path on the bus
-    sessions: HashMap<OwnedObjectPath, Arc<Session>>,
+    service: Option<Service>,
     session_index: Arc<RwLock<u32>>,
```

Comment:
Do we know what else that could be added here to the service manager ? Because it feels like we can just completely get rid of it now.

Reply:
Ha ha :)

So, we're going to pass Service instances instead of ServiceManagers to the other parts right?

Reply:
Yes